### ISSUE TITLE
joy-react-table npm

### ISSUE THREAD

**Issue by KevinVandy** [noguntuberu/react-datatable#2]
Hi,

This might be a weird message, but I wanted to inquire about the NPM package name "joy-react-table" and whether or not you'd be willing to sell it. 

I'm the creator of Material React Table: https://v2.material-react-table.com/
And I am about to create a forked project based on MUI's new "Joy UI" library: https://mui.com/joy-ui/getting-started/

I'd like to be able to name my project "Joy React Table", but if you're not willing to part with the name, it's all good. Just thought I'd reach out and give it a shot.

**Comment by noguntuberu** [comment]
Hi Kevin,

Would this project be open-source?

**Comment by KevinVandy** [comment]
> Hi Kevin,
> 
> Would this project be open-source?

Yes, it would be 100% free and open source with the MIT license. No premium, paid, or pro features either.

The new library I'm making would be almost a clone of my existing 2 libraries:
https://www.material-react-table.com/about#license
https://www.mantine-react-table.com/about#license

And if you are using `joy-react-table` for existing projects that require downloading it from NPM, don't worry, I'll just be using a back-up package name. We might see a bump in your download numbers from accidental downloads eventually, but that should not be too big of a deal.

Edit: And actually, looking into it, there is a possibility to keep up the old v1 versions you published even after an NPM ownership transfer.

**Comment by noguntuberu** [comment]
Hi Kevin,

Since the community would before more from it, I am happy to hand the name over at no cost.

**Comment by KevinVandy** [comment]
> Hi Kevin,
> 
> Since the community would before more from it, I am happy to hand the name over at no cost.

I would be very grateful for that. Would you let me give any kind of credit to you or your company on my joy-react-table.com site that I will make?
I might start work on my project early next week, and will follow up here.
I think after a transfer, we would be able to keep your v1 versions published and I would just develop on v2. Unless they are not needed anymore by you or other projects.
